sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -155,6 +161,7 @@ def _add_argparse_args(cls, parser):
 
 def model_pcollection(argv):
   """Creating a PCollection from data in local memory."""
+  # [START model_pcollection]
```

Comment:
How about cutting the snippet from the bottom and removing:
```
 | beam.io.WriteToText(my_options.output))

result = p.run()
```
Writing and running are not part of the Java snippet.